toolchain/lower/file_context.h

```diff
@@ -99,6 +106,7 @@ class FileContext {
   auto GetTypeType() -> llvm::StructType* { return context().GetTypeType(); }
 
   auto context() -> Context& { return *context_; }
+  auto context() const -> Context& { return *context_; }
```

Comment:
Generally our `Context` types are always used mutably, so this seems surprising to me. And it's not const-correct either. Can we remove the need for this?

Reply:
Ah, sure: bf2af30d2a6a438421d594db76d1f137873570f1